src/routes/user.js

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import bodyParser from 'body-parser'
+import express from 'express'
 import { Router } from 'express'
```

Comment:
Nit: might as well replace this use with `express.Router`.

```suggestion
```

Reply:
👍 https://github.com/mozilla/blurts-server/pull/3004/commits/98c7ea21d6cff07692696ea24e4cbd3af7e83c8d